Source/missiles.cpp

```diff
@@ -1177,7 +1177,7 @@ bool PlayerMHit(int pnum, Monster *monster, int dist, int mind, int maxd, missil
 		dam = player._pHitPoints / 3;
 	} else {
 		if (!shift) {
-			dam = (mind << 6) + GenerateRnd((maxd - mind + 1) << 6);
+			dam = (mind << 6) + (GenerateRnd(maxd - mind + 1) << 6);

```

Comment:
```suggestion
			dam = (mind + (GenerateRnd(maxd - mind + 1)) << 6;

```

Reply:
Unless you also want the variables renamed here, this looks to me like it should be resolved.